CHANGES.rst

```diff
@@ -6,6 +6,11 @@ associations
 
 - Update diagrams to change sloper to detector1. [#4986]
 
+coron3
```

Comment:
Since this is a pipeline module change, this should go under a "pipeline" heading and then referenced as "calwebb_coron3".